local.css

```diff
@@ -3,9 +3,13 @@
  * Originally created by Devit Schizoper (01.12.2006)
  */
 
+font-face {
+    font-family: "ProdigySans";
+    font-weight: normal;
+    src: url(/fonts/ProdigySans-Regular.ttf) format("truetype");
+}
+
 body {
-  font-family: Verdana, Arial, sans-serif;
```

Comment:
i expect that bundled OTF font to be used here or so

Reply:
(or prolly setting that --bs-font-sans-serif var to that value of bundled font)

Reply:
It's used in line 68, for `h1` headings only.

Reply:
then it's not alright, is better to use same font family, just different weight and/or combining sans/serif/mono variants